File: leetcode code/991-broken-calculator/991-broken-calculator.java
class Solution {
    public int brokenCalc(int startValue, int target) {
        int ans = 0;
        while (startValue < target){
            ans++;
            if(target%2!=0){
                target += 1;
            }
            else{
                target /= 2;
            }
        }
        return ans + startValue - target;
    }
}

File: leetcode code/991-broken-calculator/991-broken-calculator.py
class Solution:
    def brokenCalc(self, startValue: int, target: int) -> int:
        ctr = 0
        while startValue < target:
            ctr += 1
            if target%2:
                target += 1
            else:
                target //= 2
        return ctr+startValue - target